core/src/main/java/org/apache/accumulo/testing/core/randomwalk/concurrent/Config.java

```diff
@@ -190,10 +186,8 @@ private void changeTableSetting(RandomDataGenerator random, State state, RandWal
     try {
       env.getAccumuloConnector().tableOperations().setProperty(table, setting.property.getKey(), "" + newValue);
     } catch (AccumuloException ex) {
-      if (ex.getCause() instanceof ThriftTableOperationException) {
```

Comment:
How did you find what exception to replace the cause with?

Reply:
Assuming this is the correct spot, I followed what the setProperty() method in TableOperationsImpl was doing... https://github.com/apache/accumulo/blob/master/core/src/main/java/org/apache/accumulo/core/client/impl/TableOperationsImpl.java#L897